examples/with-plugins/plugins/my-plugin.ts

```diff
@@ -0,0 +1,9 @@
+import { defineNuxtPlugin } from '#imports'
+
+export default defineNuxtPlugin(() => {
```

Comment:
Main point of using plugin entry points is availability of `nuxtApp` context:

```suggestion
export default defineNuxtPlugin((nuxtApp) => {
```

Reply:
I can't add it and not use it since the CI Lint fails [1fc3403](https://github.com/nuxt/framework/pull/2855/commits/1fc3403da496e4ae7d403418d0cc8794826cfa3a)

Reply:
I could eventually write something like:
```js
export default defineNuxtPlugin((/* nuxtApp */) => {
```